.github/workflows/build-images.yml

```diff
@@ -51,6 +51,16 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: chromatichq/${{ matrix.service }}:${{ matrix.php }}
 
+      - name: Notify slack on failure
+        if: failure()
```

Comment:
I assume we don’t want failures on PR builds to be reported. Since we’re expecting some builds to fail, I’m using this as an opportunity to validate that this works as expected. Once that is confirmed, I’ll figure out how this needs to be modified to check for failures _and_ that this is not a pull request.

Reply:
Updated in 1067388.